packages/bierzo-wallet/src/routes/registerName/index.tsx

```diff
@@ -99,6 +100,52 @@ const validate = async (values: object): Promise<object> => {
   return errors;
 };
 
+function getChainAddressPairsFromValues(
+  values: FormValues,
+  addresses: readonly ChainAddressPairWithName[],
+): readonly ChainAddressPairWithName[] {
+  const chainAddressMap: Map<string, Partial<ChainAddressPairWithName>> = new Map<
+    string,
+    Partial<ChainAddressPairWithName>
+  >();
+  Object.keys(values).forEach(key => {
+    const index = key.substr(0, fieldValueIdxLength);
+    let pair = chainAddressMap.get(index);
+    if (!pair) {
+      pair = {};
+    }
+
+    const type = key.substr(fieldValueIdxLength + 1);
+    switch (type) {
+      case addressValueField:
+        pair = { ...pair, address: values[key] as Address };
+        break;
+      case blockchainValueField:
+        // eslint-disable-next-line no-case-declarations
```

Comment:
Instead of this exception, please wrap code in `{ .... }` as in the example from https://eslint.org/docs/rules/no-case-declarations. Usually this was auto-fixed ... Did it not work?

```
switch (foo) {
    // The following case clauses are wrapped into blocks using brackets
    case 1: {
        let x = 1;
        break;
    }
    case 2: {
        const y = 2;
        break;
    }
    case 3: {
        function f() {}
        break;
    }
    case 4:
        // Declarations using var without brackets are valid due to function-scope hoisting
        var z = 4;
        break;
    default: {
        class C {}
    }
}
```

Reply:
No, it is not